datafusion/core/src/optimizer/filter_push_down.rs

```diff
@@ -894,27 +889,6 @@ mod tests {
         Ok(())
     }
 
-    #[test]
-    fn union_all_with_alias() -> Result<()> {
```

Comment:
Rather than remove this test, what about updating it to use `LogicalPlanBuilder::union` instead?

Reply:
The test is specifically testing for creating a union with an alias, which we no longer support

Reply:
On second thoughts, the test is about using a filter with an alias ... :thinking: ... ok I will look into this more